index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <title>Danny Rand | About Me</title>
+      <link rel="stylesheet" href="css/styles.css">
+  </head>
+
+  <body>
+    <nav>
+      <strong>About Me</strong>|
```

Comment:
@tjafari instead of the strong tag, try using a `span` tag. Then style that span tag through css.